### PR TITLE
COMMERCE-1065 | 7.1.x-next

### PR DESCRIPTION
**Pull request by ZoltanTakacs**
[Commerce API Scrum board](https://issues.liferay.com/secure/RapidBoard.jspa?rapidView=3928)
----
Ticket: https://issues.liferay.com/browse/COMMERCE-1065

/cc @matijapetanjek

### CODE REVIEW

**Comment by ZoltanTakacs** [comment]
ci:test

**Comment by ZoltanTakacs** [comment]
ci:retest

**Comment by marco-leo** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
updated 7.1.x-next in private

File: commerce-batch/commerce-batch-service/service.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		<column name="startTime" type="Date" />
+		<column name="endTime" type="Date" />
+		<column name="status" type="String" />
+		<column name="exitCode" type="String" />
+		<column name="exitMessage" type="String" />
+
+		<!-- Finder methods -->
+
+	</entity>
+	<entity local-service="true" name="CommerceBatchJobInstance" remote-service="true">
```
Comment (marco-leo) [inline commerce-batch/commerce-batch-service/service.xml:30]: CommerceBatchEntry

Comment (ZoltanTakacs) [reply commerce-batch/commerce-batch-service/service.xml:30]: @matijapetanjek as per our discussion, please update the PR to only have one table `CommerceBatchEntry` and merge the related fields in it: https://docs.google.com/document/d/1gCUYoc5wHTTOaSVArIePHbSrBdRsixmEux3l6dSvEuE/edit#heading=h.xq63w7dtts9b

**Comment by ZoltanTakacs** [comment]
ci:close
Closing this one as it will be updated.

**Pull request closed without merging**